src/bacnet/basic/object/nc.c

```diff
@@ -486,6 +486,91 @@ void Notification_Class_Get_Priorities(
         pPriorityArray[i] = CurrentNotify->Priority[i];
 }
 
+bool Notification_Class_Get_Recipient_List(
+    uint32_t Object_Instance, BACNET_DESTINATION *pRecipientList)
+{
+  NOTIFICATION_CLASS_INFO *CurrentNotify;
+  uint32_t object_index;
+
+  object_index = Notification_Class_Instance_To_Index(Object_Instance);
+
+  if (object_index < MAX_NOTIFICATION_CLASSES) {
+    CurrentNotify = &NC_Info[object_index];
+    int i;
```

Comment:
Just for consistency I would use "int32_t" type or similar instead of the "int" primitive. (General comment for the whole file, so maybe for future improvement, get rid of the primive data types ?)

Reply:
Aye - we try not to be consistent with the upstream.

Reply:
I believe it is also in the elusive SE C coding standard document